micro-rdk/src/esp32/conn/network.rs

```diff
@@ -268,12 +270,31 @@ impl Esp32WifiNetwork {
         &self,
         mut networks: Vec<NetworkSetting>,
     ) -> Result<(), WifiManagerError> {
-        // TODO(RSDK-10184): scan available networks first
-        // attempt to connect only if available
+        log::info!("initializing wifi module");
+        // `set_station_mode` ends by calling `connect`, which will error here
+        let _err = self
+            .set_station_mode(NetworkSetting::new("".to_string(), "".to_string(), 0))
+            .await;
+
+        log::info!("scanning available networks");
```

Comment:
Do you want to log the available networks afterwards?

Reply:
No, I think I agree, having considered it more and reading your comment, that it is likely too chatty.